reconcile/ocm_upgrade_scheduler.py

```diff
@@ -69,14 +74,22 @@ def fetch_desired_state(
         if not ocm.is_ready(cluster_name):
             # cluster has been deleted in OCM or is not ready yet
             continue
-        spec = ocm.clusters[cluster_name].spec
-        upgrade_policy["current_version"] = spec.version
-        upgrade_policy["channel"] = spec.channel
         # Replace sector names by their related OCM Sector object, including dependencies
         sector_name = upgrade_policy["conditions"].get("sector")
         if sector_name:
             upgrade_policy["conditions"]["sector"] = ocm.sectors[sector_name]
-        desired_state.append(upgrade_policy)
+        if not addons:
+            spec = ocm.clusters[cluster_name].spec
+            upgrade_policy["current_version"] = spec.version
+            upgrade_policy["channel"] = spec.channel
+            desired_state.append(upgrade_policy)
+            continue
```

Comment:
forgive me that the only thing i have to say is a stupid nit: 
would you consider putting the `addons` code block into an `else` for better readability as you did it in `fetch_current_state`?

Reply:
Wanted to avoid longer lines :) but I'll do that

Reply:
done in [894376f](https://github.com/app-sre/qontract-reconcile/pull/3120/commits/894376fda09fa507556460399447b8ddb089f0c5)